lib/elixir/src/elixir_erl_pass.erl

```diff
@@ -66,6 +66,9 @@ translate({'__block__', Meta, Args}, S) when is_list(Args) ->
 translate({'__CALLER__', Meta, Atom}, S) when is_atom(Atom) ->
   {{var, ?ann(Meta), '__CALLER__'}, S#elixir_erl{caller=true}};
 
+translate({'__STACKTRACE__', Meta, Atom}, S = #elixir_erl{stacktrace={Var, _}}) when is_atom(Atom) ->
+  {{var, ?ann(Meta), Var}, S#elixir_erl{stacktrace={Var, true}}};
```

Comment:
I think we can do like caller and simply use the `__STACKTRACE__` variable. This way the stacktrace field only keeps a boolean.

Reply:
We can't since `try` (and stacktraces) could be nested, unlike `__CALLER__`.